test/Transforms/addf_spirv.mlir

```diff
@@ -0,0 +1,53 @@
+// RUN: imex-opt -serialize-spirv %s | FileCheck %s
```

Comment:
* Rename file to `serialize-spirv.mlir`
* This test should be added to `main` first and only after that ported to `refactor`

Reply:
thanks for pointing out, recommitted, pls check

Reply:
Things don't have to go through main to be added to refactor